src/lib/Scenes/Artwork/Components/__stories__/ArtworkAvailability.story.tsx

```diff
@@ -3,5 +3,12 @@ import React from "react"
 import { ArtworkAvailability } from "../ArtworkAvailability"
 
 storiesOf("Artwork/Components").add("Availability", () => {
-  return <ArtworkAvailability artwork={{ id: "Blah", _id: "HI", availability: "On Loan", " $refType": null }} />
+  return (
```

Comment:
Was getting a TS error on this line because we're not actually importing any of the other fields on the relay query.